### DISCUSSION TITLE
nonebot1匹配图片消息

### DISCUSSION THREAD

**Discussion by lztlzt-1** [nonebot/discussions#365] [问答]
最近想试着收到图片信息时回复消息，尝试了在on_command里通过正则匹配控制面板给出的图片格式，但好像无法成功匹配，请问有没有什么办法

**Comment by yanyongyu** [comment]
你可以使用 `on_natural_language`

**Reply by lztlzt-1** [reply]
想请问下用@on_natural_language匹配什么呢，我尝试`@on_natural_language(keywords={'[CQ:image'})`，以及`@on_natural_language(keywords={'gchat.qpic.cn'})·似乎并没有作用
